src/ReinforcementLearningDatasets/src/atari/npzreadarray.jl

```diff
@@ -0,0 +1,188 @@
+# Credits: https://github.com/fhs/NPZ.jl
```

Comment:
Why do we need this file?

Reply:
The dataset is stored in numpy array files. I did not know how to extract it. So, NPZ.jl provides a neat package to manage this extraction, but it supports input as the string of the file path(.npy or .npz) or IO. It does work on GZStream but is not supported. 

So this is just a temporary workaround. I am planning to make a PR in that repo to make it work by just importing NPZ.jl

Reply:
It defines readnpzarray function that is used. I did not put all the functions in, so it is not working. Will add everything and it will show in the tests.

Reply:
so based on your description, what we need is to open a gzip file as IO stream and then use NPZ.jl to extract data, right?
If so I remember there's a package to do the uncompression

Reply:
NPZ will take care of just the reading part and not the decompression.

So the plan here is to decompress using gzipopen and use NPZ to read from the stream.

Reply:
I am looking at TranscodingStreams.jl for now.

Reply:
Works